common/models/meetings.js

```diff
@@ -10,6 +10,7 @@ const { creatCsvFile } = require('download-csv');
 const { meetingDates } = require('../../server/common/dates');
 const changeEmail = require('../../server/changeEmail');
 const changeDateTime = require('../../server/changeDateTime');
+const sendMailWithAttched = require('../../server/emailsWithAttached');
```

Comment:
sendMailWithAtt**a**ched

Also, I thought that the whole business logic would be in sendEmail (so you won't need to care about it) but you can do it like that

Reply:
true, I moved everything to a single file and imported from it

Reply:
didn't mean the same file - I meant the same function :)

Reply:
Hi I've pushed all my changes, I will put both functions in the  same function soon.
Currently sendMailWithAttached works and sends an attached CSV file, however the file does not contain the requested data but rather a long string which I suspect is the product of meeting.js line 1807 
content: Buffer.from(creatCsvFile(datas, columns)).toString("base64"),
nodeMailer requires that attachments.content will contain the file format (csv), I tried changing it but it still didn't work
What should I do?

Reply:
So both mailing function are still separate, however I've created a new function in the newEmail file that's called sendEmail, the function checks whether the email contain attached files and operates the required mailing function accordingly (this function is the the one that is imported to meetings.js .
Also just in case I've added a try/catch block for sendMailWithAttched.
Please review my last commit and let me know if there are anything more I should improve (Would it be better practice to write newEmail in OOP way?) 
If there isn't I think we can merge and ask Assaf to upload the new version.